presto-parser/src/main/java/com/facebook/presto/sql/tree/AstVisitor.java

```diff
@@ -352,6 +352,11 @@ protected R visitArithmeticUnary(ArithmeticUnaryExpression node, C context)
         return visitExpression(node, context);
     }
 
+    protected R visitTableVersion(TableVersionExpression node, C context)
+    {
+        return visitExpression(node, context);
+    }
+
```

Comment:
Should we also implement this method in ExpressionFormatter?

Reply:
Done